server/routes/journey/intercept/unknownPreRecallSentenceTypeController.ts

```diff
@@ -46,14 +46,22 @@ export default class UnknownPreRecallSentenceTypeController implements Controlle
       recallId,
       RecallJourneyUrls.manualJourneyStart.name,
     )
-    const continueLink = `${config.urls.remandAndSentencing}/person/${nomsId}`
+
+    const sentenceUuids = Array.from(
+      unknownPreRecallCourtCases.flatMap(courtCase => courtCase.sentences.map(sentence => sentence.sentenceUuid)),
+    )
+    const params = new URLSearchParams()
+    sentenceUuids.forEach(uuid => params.append('sentenceUuids', uuid))
+
+    const continueLink = `${config.urls.remandAndSentencing}/person/${nomsId}/unknown-recall-sentence?${params.toString()}`
```

Comment:
on the receiving end (RAS) they should be able to extract from the query directly into an array like this
`const sentenceUuids = req.query.sentenceUuids as string[]`